Finished the API

Weeezzt opened the pull request:
Filled the API with ten new, ten available and ten movies for kids


<img width="872" alt="Screenshot 2023-12-22 at 17 27 15" src="https://github.com/Kookiie88/A4_biografSajt/assets/144351044/87920c0f-7870-46f6-b855-0259282d8673">

Kookiie88 reviewed (approved):
Great work all the objects seem to have the correct properties and no values are missing.

Kookiie88 commented:
I would suggest too add the month that the movie is also released, 1 year a pretty big timespan.

AnnaLenaO commented on movies.json:
```diff
@@ -1,16 +1,365 @@
+            "release_year": 2014,
+            "genre": ["Äventyr", "Drama"],
+            "actors": ["Matthew McConaughey", "Anne Hathaway"],
+            "description": "",
```
Please add a description if necessary.

Weeezzt replied:
Done

AnnaLenaO commented on movies.json:
```diff
@@ -1,16 +1,365 @@
+            "release_year": 2017,
+            "genre": ["Animerad", "Äventyr", "komedi"],
+            "actors": ["Alec Baldwin", "Steve Buscemi"],
+            "description": "",
```
Please add a description if necessary.

Weeezzt replied:
Done

AnnaLenaO commented on movies.json:
```diff
@@ -1,16 +1,365 @@
+            "director": "Robert Zemeckis",
+            "release_year": 2004,
+            "genre": ["Animerad", "Äventyr"],
+            "actors": [""],
```
Please add actors if necessary.

Weeezzt replied:
Fixed! :)

Weeezzt replied:
Fixed the missing info. There are 10 upcoming movies but 5 that has their world premiere after new year

AnnaLenaO reviewed (commented):
Great work with all 10 properties. There are three missing values that can be added if necessary. 
According to the PBI it sholud be at least 10 upcoming, 10 child moovies & 10 available moovies. Now there are 5 moovies with release year 2024. If it is important for us with at least 10 upcoming , please change. Otherwise it can be like this :)

Weeezzt commented:
I've added a new key premiere to all movies with "new": true. It has a value of "Premiär 6 Januari" for example. I've also fixed some missing info that Anna-Lena found. There are 10 new/upcoming movies so i wont be changing or adding any more movies to the API. Please have another look at the changes and I'll listen if theres new feedback :)

EmilioSanchezLernia reviewed (approved):
Looks good, very informative API with plenty of opportunity to filter.
